shared/main.tf

```diff
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+resource "aws_vpc" "devops-vpc" {
```

Comment:
Let's just keep one VPC defined in `main` per my comment above (and using pre-existing module from AWS). Let's remove this

Reply:
Good one, removed in [346e992](https://github.com/nvminhtue/tyson_devops_ic/pull/18/commits/346e992d89be49ee9dd223a249564b2c6684a74c)